app/assets/stylesheets/alchemy/sitemap.scss

```diff
@@ -1,3 +1,107 @@
+#sitemap {
+
+  &.list {
+    padding: 16px 12px;
+  }
+
+  &.sorting {
+
+    .page.icon { cursor: move }
+  }
+
+  .page.icon {
+    float: left;
+    position: relative;
+  }
+
+  .locked .page.icon {
+    background-position: -32px -40px;
+  }
+
+  .legend {
+    color: #666;
+    line-height: 2em;
+
+    .node_url { padding-right: 84px }
+    .node_name { margin-left: 32px }
+  }
+
+  .root_node {
+    font-weight: bold;
+    margin-bottom: 1em;
+
+    &:hover {
+      .sitemap_right_tools { display: block }
+    }
+
+    .sitemap_right_tools { display: none }
+  }
+
+  .page_folder {
+    @extend .icon;
+    float: left;
+    margin-left: -21px;
+    margin-top: 1px;
+
+    &.collapsed { @extend .icon.expand.small }
+    &.folded { @extend .icon.collapse.small }
+  }
+
+  ul {
+    margin: 0;
+    padding: 0;
+  }
+
+  li {
+    padding-left: 0;
+
+    .selected_page a {
+      background-color: $linked-color;
+      color: $dark-gray;
+    }
+
+    li {
+      padding-left: 22px;
+      padding-right: 0;
+    }
+  }
+
+  .node_type, .node_url {
+    color: #666;
+    float: right;
+    padding-right: 8px;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+  }
+
+  .node_type { width: 140px }
```

Comment:
Selector `node_type` should be written in lowercase with hyphens<br>Declaration should be terminated by a semicolon